### PR TITLE
Env - Artillery

### PR DESCRIPTION
**Pull request by Kinvert**

### CODE REVIEW

**Comment by Kinvert** [comment]
It works. Trains in a couple minutes. Still some improvements to be made.

**Comment by Kinvert** [comment]
This trains great in around 2 minutes. It works great in eval but for some reason exporting the model and running in ./artillery is much less impressive. I think it's in the model itself. They're sent the same observations and different logits come out.

**Comment by Kinvert** [comment]
ok so besides ./artillery performing worse than eval, the only other actual issue I know of with this is that maybe half the time the training will diverge. So sometimes I have to run train twice. This is basically ready besides those things for the most part. So in the mean time I'm gunna start making a 3D version of this.

**Comment by Kinvert** [comment]
ok I don't know why yet but at [fd1a788](https://github.com/PufferAI/PufferLib/pull/307/commits/fd1a78856a8c3ce557fae9e12b5236fa9ab3aa7e) ./artillery is performing closer to eval. It still shoots generally behind the target a bit. This is after trying to implement some of what BET suggested.

What seemed to fix it was setting --vec.num_workers 1. I don't know why it worked yet.
